src/Tutorial/PatientBrowser/index.html

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>FHIR Patient browser sample app</title>
+    <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.0/js/msal.js"></script>
+    <style>
+    
+    @font-face {
```

Comment:
Should we just use bootstrap (loaded from CDN) style and avoid custom styling alltogether?